### PR TITLE
Cloud Function for AVRO import to BQ

### PR DESCRIPTION
**Pull request by mkahn5**
This tutorial demonstrates how to use a Cloud Function to automate the import of a CSV or AVRO in Google Cloud Storage to BigQuery.

- [x] Tech writing / styleguide review - @labadav 
- [x] Formatting review for publishing - @jmdobry

### CODE REVIEW

**Comment by mkahn5** [comment]
Bump

**Comment by mkahn5** [comment]
@labadav @jmdobry Checking status of this, thanks

File: tutorials/cloud-functions-avro-import-bq/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        unzip gcf_gcs.zip
+
+1.  Update `index.js` with your `projectId` and `datasetId` (destination).
+1.  Update `install.sh` `--trigger-resource` flag with your source Cloud Storage
+    bucket (replace avro-import-source)
+1.  Run the function: 
+
+        ./install.sh
+
+1.  Verify the function is running in the [GCP console][console].
```
Comment (labadav) [inline tutorials/cloud-functions-avro-import-bq/index.md]: **C**onsole

File: tutorials/cloud-functions-avro-import-bq/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+1.  Upload an avro to the source Cloud Storage bucket you specified in `install.sh`.
```
Comment (labadav) [inline tutorials/cloud-functions-avro-import-bq/index.md]: AVRO file

File: tutorials/cloud-functions-avro-import-bq/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+[console]: https://console.cloud.google.com/functions/
+
+This cloud function should deploy and wait for new objects to be finalized on
+the source Cloud Storage bucket. Once a new AVRO is uploaded to the source it
```
Comment (labadav) [inline tutorials/cloud-functions-avro-import-bq/index.md]: AVRO file

File: tutorials/cloud-functions-avro-import-bq/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+will use the BQ API to load the new dataset into a new table.
```
Comment (labadav) [inline tutorials/cloud-functions-avro-import-bq/index.md]: BigQuery

File: tutorials/cloud-functions-avro-import-bq/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+Note: This will only create a new table for each new import. You may need to
```
Comment (labadav) [inline tutorials/cloud-functions-avro-import-bq/index.md]: You **might**

**Comment by mkahn5** [comment]
@labadav thanks Ivan. Is anything required on my end to finalize this PR?

**Comment by mkahn5** [comment]
Can this PR be completed? @jmdobry @labadav

**Comment by mkahn5** [comment]
Checking on status of this @jmdobry @labadav 
Thanks

**Comment by labadav** [comment]
@mkahn5 Hi Mike. Can you make the changes that I pointed out in the comments? I'll approve then.

**Comment by mkahn5** [comment]
@labadav Thanks Ivan, just made a [commit](https://github.com/GoogleCloudPlatform/community/pull/430/commits/84b2a9d838e11dc83e21e7b92ee9d8ecdcc15a74) with the changes completed.

**Pull request merged**